### ISSUE TITLE
ARM MID hardware failures catchall

### ISSUE THREAD

**Issue by bobbybudnick** [bobbybudnick/SFS-Mobile-Internet-Device#38]
New Issue Report

Description:
failure investigation 1
still seeing flickering after device warms up
test with power supply on usb-c connector at 7.5v - flickering
test with power supply on usb-c connector at 11.2v - flickering worse
we can see that unfortunately using the usb-c does not help much if at all
again it does not seem to affect stability
youtube seems to be worst case scenario for flickering
one radical fix might be to tap 5v from gpio and wire to screen micro usb power
gpio power boost made no difference so this means device 5v conversion is poor
for thin devices one with a flickering screen is better than a slow one like model zero
ultimately more investigation is needed                                              

failure investigation 2
an exploded device was built with identical components and works fine
the original device will flicker even when using ethernet
possible reasons for failure
heat from tighter design - the clone has less heatsinks so this is unlikely
check screen settings and match - negative because exploded device works with screen
defective screen - negative because it works with exploded device and ac power supply
defective stock usb cable - negative because it works with exploded device
defective hub causing higher 5v load - negative because it works with exploded device
one usb port may be stronger than the other - exploded device works fine both ways
wear and tear/difference between individual vim 3 - definitely the sbc itself
there is only the very slightest flicker occasionally with the exploded device

Explanation:
The individual VIM 3 is defective and not all VIM 3 units are defective.

Workaround:
To summarize ARM MID usage - at least on VIM 3 a power switch for components is not needed because these are powered off when sleeping. This makes interference naturally less of a problem. For now it seems ok to have the screen directly connected to one port on VIM 3 and on the other port a hub connected to the cellular device and another device like a keyboard or microphone.

-----

Mid Level Issue Report

Description: 4.3" screen flickers when watching a youtube video
when volume is increased and cores enabled the screen flickering is more noticeable
should not be reaching the battery current limit because current is well below 1 amp
does not seem to affect stability but the flickering is too unusable
Explanation:
control test with battery and right angle hub switched to left angle to expose usb-c
works fine

test with power supply on usb-c connector at 7.5v and 11.2v
voltage boosted slightly from average to account for wiring voltage drop
not necessary now but good to use in the future

one fix might be to put the usb-c connector in parallel with the vin connector
especially given the usb-c connector is not useful for legacy devices as it is in parallel with the vin
not necessary now but good to use in the future

3s may be more reliable
it seems there were not many power problems earlier at least with the 4.3" screen
the alpha and delta do not generally have brownout issues and they have a 3s system

Workaround:
remove all power draining accessories
replace hub
attach usb-c connector in parallel with battery wiring
change from a 2s battery system to a 3s system

Models affected: 38

-----

Earlier Issue Report

Description: At KDE login power usage peaks and device resets
Explanation: The 5v current limit is reached due to the additional power draw of the 5" screen
Workaround: Turn off screen at login time
Models affected: 30

Fixed: closed because ARM devices are not used anymore.
